cli/flox/src/utils/init/logger.rs

```diff
@@ -89,9 +90,25 @@ pub fn create_registry_and_filter_reload_handle() -> (
         }));
 
     // Tracing layer that handles all other logs.
+    //
+    // Span data is added to _all_ events within them, and "stack" if multiple spans are active.
+    // While the JSON formatter seems to support to suppress this span information,
+    // the same is not possible with either of the other builtin formatters.
+    //
+    // An existing issue on that upstream appears not to have active development:
+    // <https://github.com/tokio-rs/tracing/issues/3254>
+    //
+    // What is possible however is to filter out the _"progress" fields_,
+    // so that spans are still printed but we don't repeat the messages.
+    // That is using the `FilteredFormatFields` utility from `tracing_indicative`,
+    // which is a visitor implementation that just dropts fields based on a filter function,
+    // here: a test for the field name "progress".
     let log_layer = tracing_subscriber::fmt::layer()
         .with_writer(writer.clone())
         .with_ansi(use_colors)
+        .map_fmt_fields(|format| {
+            FilteredFormatFields::new(format, |field| field.name() != "progress")
```

Comment:
(non-blocking) Pulling `"progress"` out to a const would give a clearer indication that it has a special meaning across several places.

Reply:
not sure if the actual spinners/spans can use a const 🤔